todo-list-rbeckham/src/App.js

```diff
@@ -1,19 +1,25 @@
-import React, { useState } from "react";
+import React, { Component, useState } from "react";
 
 import "./App.css";
 import NewToDo from "./components/NewToDo";
-
-function App() {
+import ToDoList from "./components/ToDoList";
+function App(props) {
   const [tasks, setTasks] = useState([]);
-  const addTaskHandler = (tasks) => {
+  const addTaskHandler = (task) => {
     setTasks((prevTasks) => {
-      return [...prevTasks, tasks];
+      return [...prevTasks, task];
     });
   };
-  console.log(tasks);
+  const updateTasksHandler = (id) => {
+    const newTasks = tasks.filter((task) => task.id !== id);
+    setTasks(newTasks);
+  };
   return (
     <div>
       <NewToDo onAddTask={addTaskHandler}></NewToDo>
+      {tasks.length !== 0 && (
```

Comment:
Should save this as a variable near the top to reference this. Also `tasks.length` is the same as `tasks.length !== 0`. So no need to have the equals signs. It is  a falsy value so react will render it if there are tasks.